src/index.js

```diff
@@ -2,22 +2,83 @@ import 'lodash';
 import './style.css';
 import './queries.css';
 
-const addScore = () => {
-  let name = document.querySelector('.name').value;
-  let score = document.querySelector('.score').value;
+async function getData(url) {
+  const myRequest = new Request(url, {
+    method: 'GET',
```

Comment:
- [ ] Consider using arrow function instead of function keyword as arrow functions allow us to write shorter function syntax and ensure we're following ES6 standard. 😉. Check out this [link](https://www.w3schools.com/js/js_arrow_function.asp) to learn more about arrow functions